easybuild/easyconfigs/n/ncurses/ncurses-6.0.eb

```diff
@@ -14,6 +14,9 @@ toolchainopts = {'optarch': True, 'pic': True}
 source_urls = [GNU_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
+# ncurses-6.0 builds fail with `mawk`, so force use of `gawk`
+# see http://lists.gnu.org/archive/html/bug-ncurses/2015-08/msg00013.html
+preconfigopts = "env AWK=/usr/bin/gawk "
```

Comment:
What other `awk`'s are available on Linux? I only know of `mawk` and `gawk` -- and ncurse won't compile with `mawk`.